search/intermediate_result.hpp

```diff
@@ -74,59 +72,30 @@ class PreResult2
     m_info = forward<TInfo>(info);
   }
 
-  /// @param[in]  infoGetter Need to get region for result.
-  /// @param[in]  pCat    Categories need to display readable type string.
-  /// @param[in]  pTypes  Set of preffered types that match input tokens by categories.
-  /// @param[in]  lang    Current system language.
-  /// @param[in]  coder   May be nullptr - no need to calculate address.
-  Result GenerateFinalResult(storage::CountryInfoGetter const & infoGetter,
-                             CategoriesHolder const * pCat, set<uint32_t> const * pTypes,
-                             int8_t locale, ReverseGeocoder const * coder) const;
-
-  /// Filter equal features for different mwm's.
-  class StrictEqualF
-  {
-  public:
-    StrictEqualF(PreResult2 const & r, double const epsMeters);
-
-    bool operator()(PreResult2 const & r) const;
-
-  private:
-    PreResult2 const & m_r;
-    double const m_epsMeters;
-  };
-
-  /// To filter equal linear objects.
-  //@{
-  struct LessLinearTypesF
-  {
-    bool operator() (PreResult2 const & r1, PreResult2 const & r2) const;
-  };
-  class EqualLinearTypesF
-  {
-  public:
-    bool operator() (PreResult2 const & r1, PreResult2 const & r2) const;
-  };
-  //@}
-
   string DebugPrint() const;
 
   bool IsStreet() const;
 
   inline FeatureID const & GetID() const { return m_id; }
   inline string const & GetName() const { return m_str; }
   inline feature::TypesHolder const & GetTypes() const { return m_types; }
+  inline Type const & GetResultType() const { return m_resultType; }
```

Comment:
Actually it's worth to remove "inline" keyword here.

Reply:
done